test/parallel/test-stringbytes-external.js

```diff
@@ -44,9 +44,9 @@ assert.strictEqual(b[1], 0);
 assert.strictEqual(ucs2_control, c);
 
 // now create big strings
-const size = 1 + (1 << 20);
-write_str = Array(size).join(write_str);
-ucs2_control = Array(size).join(ucs2_control);
+const size = 1 << 20;
```

Comment:
When you do the `Array.join()` it ends up as one less than `size`, so when you move to repeat you have to decrease size by one to get the same string length.

```js
> Array(3).join('a')
'aa'
> 'a'.repeat(3)
'aaa'
```